pkg/config/config.go

```diff
@@ -30,70 +30,88 @@ var OutFormats = []string{
 }
 
 type ExcludePattern struct {
+	ID      string
 	Pattern string
 	Linter  string
 	Why     string
 }
 
 var DefaultExcludePatterns = []ExcludePattern{
 	{
+		ID: "EXC0001",
 		Pattern: "Error return value of .((os\\.)?std(out|err)\\..*|.*Close" +
 			"|.*Flush|os\\.Remove(All)?|.*printf?|os\\.(Un)?Setenv). is not checked",
 		Linter: "errcheck",
 		Why:    "Almost all programs ignore errors on these functions and in most cases it's ok",
 	},
 	{
+		ID: "EXC0002",
 		Pattern: "(comment on exported (method|function|type|const)|" +
 			"should have( a package)? comment|comment should be of the form)",
 		Linter: "golint",
 		Why:    "Annoying issue about not having a comment. The rare codebase has such comments",
 	},
 	{
+		ID:      "EXC0003",
 		Pattern: "func name will be used as test\\.Test.* by other packages, and that stutters; consider calling this",
 		Linter:  "golint",
 		Why:     "False positive when tests are defined in package 'test'",
 	},
 	{
+		ID:      "EXC0004",
 		Pattern: "(possible misuse of unsafe.Pointer|should have signature)",
 		Linter:  "govet",
 		Why:     "Common false positives",
 	},
 	{
+		ID:      "EXC0005",
 		Pattern: "ineffective break statement. Did you mean to break out of the outer loop",
 		Linter:  "staticcheck",
 		Why:     "Developers tend to write in C-style with an explicit 'break' in a 'switch', so it's ok to ignore",
 	},
 	{
+		ID:      "EXC0006",
 		Pattern: "Use of unsafe calls should be audited",
 		Linter:  "gosec",
 		Why:     "Too many false-positives on 'unsafe' usage",
 	},
 	{
+		ID:      "EXC0007",
 		Pattern: "Subprocess launch(ed with variable|ing should be audited)",
 		Linter:  "gosec",
 		Why:     "Too many false-positives for parametrized shell calls",
 	},
 	{
+		ID:      "EXC0008",
 		Pattern: "G104",
 		Linter:  "gosec",
 		Why:     "Duplicated errcheck checks",
 	},
 	{
+		ID:      "EXC0009",
 		Pattern: "(Expect directory permissions to be 0750 or less|Expect file permissions to be 0600 or less)",
 		Linter:  "gosec",
 		Why:     "Too many issues in popular repos",
 	},
 	{
+		ID:      "EXC0010",
 		Pattern: "Potential file inclusion via variable",
 		Linter:  "gosec",
 		Why:     "False positive is triggered by 'src, err := ioutil.ReadFile(filename)'",
 	},
 }
 
-func GetDefaultExcludePatternsStrings() []string {
+func GetDefaultExcludePatternsStrings(include []string) []string {
```

Comment:
Can you please create a new function so we can maintain backward compatibility in API?
E..g. `GetExcludePatternsStrings(include []string)` that will be called from `GetDefaultExcludePatternsStrings()`.